phpBB/phpbb/language/language_file_helper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\language;
+
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Helper class for language file related functions
+ */
+class language_file_helper
+{
+	/**
+	 * @var string	Path to phpBB's root
+	 */
+	protected $phpbb_root_path;
+
+	/**
+	 * Constructor
+	 *
+	 * @param string	$phpbb_root_path	Path to phpBB's root
+	 */
+	public function __construct($phpbb_root_path)
+	{
+		$this->phpbb_root_path = $phpbb_root_path;
+	}
+
+	/**
+	 * Returns available languages
+	 *
+	 * @return array
+	 */
+	public function get_available_languages()
+	{
+		// Find available language packages
+		$finder = new Finder();
```

Comment:
`phpbb\finder` seems to lack the functionality I use here, also caching is not an issue here.